### PR TITLE
Tasklist overflow issue fixed

### PR DESCRIPTION
**Pull request by Bonymol-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE
https://aottech.atlassian.net/browse/FWF-1809

# Changes

Overflow issue in tasklist fixed

# How has the change been tested?

# Screenshots (if applicable)

image

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

**Review by sreehari-aot — APPROVED** [review]
LGTM!

**Review by abilpraju-aot — APPROVED** [review]
LGTM

File: forms-flow-web/src/components/ServiceFlow/ServiceFlow.scss [hunk 1]
```diff
@@ -150,6 +150,7 @@
 
 .font-size-16{
   font-size: 16px;
+  overflow-wrap: break-word;
```
Comment (shuhaib-aot) [inline forms-flow-web/src/components/ServiceFlow/ServiceFlow.scss:153]: Nice move

**Review by shuhaib-aot — COMMENTED** [review]
Did you check with if the assignee name length is too big

**Review by AbijithS-aot — APPROVED** [review]

**Review by abhilash-aot — APPROVED** [review]
css update on work break

**Pull request merged**